lib/3scale/backend/usage_limit.rb

```diff
@@ -3,7 +3,7 @@ module Backend
     class UsageLimit
       include Storable
 
-      PERIODS = Period::ALL_DESC
+      PERIODS = Period::ALL_DESC.reject { |period| period == :second }
```

Comment:
`.freeze`?

Reply:
Another option:
```ruby
Period::ALL_DESC - [:second]
```

Reply:
This
```
Period::ALL_DESC - [:second]
```
does not work. It is an array of granularity class objects created [here](https://github.com/3scale/apisonator/blob/0c290d4eb00584f1c51acee274ffa4373d080443/lib/3scale/backend/period/period.rb#L224)

But somehow (not investigated yet why)
```
Period::ALL_DESC.reject { |period| period == :second }
```
works

Reply:
Because the equality is overridden - set operations are probably looking for an implementation of `hash` which differs between different objects, so they are not seen as equal. Using `Period::Second` or `Period.from :second` would likely work.